src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -420,4 +463,43 @@ public void changeItemStatus(String itemStatus) {
   public void changeDeclaredLostDateTime(DateTime dateTime) {
     write(representation, LoanProperties.DECLARED_LOST_DATE, dateTime);
   }
+
+  private class Policies {
```

Comment:
Why is this a private class within the loan?

Reply:
because Policies class isn't used out of a loan. It just need to reduce number of parameters in the constructor of the loan

Reply:
Ok, personally, I would likely define this separately. I will resolve this comment.